homeassistant/auth/providers/command_line.py

```diff
@@ -88,9 +88,9 @@ async def async_validate_login(self, username: str, password: str) -> None:
             for _line in stdout.splitlines():
                 try:
                     line = _line.decode().lstrip()
-                    if line.startswith("#"):
+                    if line.startswith("#") or "=" not in line:
```

Comment:
Is the `except ValueError` still needed?

Reply:
If only decode can fail, then the rest should be moved out of try...except

Reply:
Done.